src/agni/client/AgniClient.java

```diff
@@ -1,13 +1,46 @@
 package agni.client;
 
+import agni.client.communication.*;
+import agni.client.receiver.*;
+import agni.client.action.*;
+import agni.client.view.*;
+
 public class AgniClient {
 
     public AgniClient() {
-        
-    }    
+
+    }
 
     public static void main(String[] args) {
-        // TODO Auto-generated method stub
+        MessageSender messageSender = new MessageSender();
+        MessageReceiver messageReceiver = new MessageReceiver();
+        MessageParser messageParser = new MessageParser() {
+        };
```

Comment:
we don't need ot actually create a `MessageParser`. The Receivers just inherit the `MessageParser`